rfcs/rework-index-updates.md

```diff
@@ -0,0 +1,60 @@
+# Summary
+
+This RFC proposes moving away from the current model of fetching new releases
+to build, moving from running [crates-index-diff] in a thread to using webhooks
+and the [crates-index] crate.
+
+# Motivation
+
+While the current approach has worked well for us so far, it has some problems:
+
+* Running the update in a cronjob every 2 minutes is wasteful, as there is
```

Comment:
nit: it's not actually a cronjob, we spawn a thread every minute.

https://github.com/rust-lang/docs.rs/blob/d6baf144a72edca646240507707b9853e39696a4/src/utils/daemon.rs#L64

Reply:
That's what happens when I write those late at night.